dallinger/command_line.py

```diff
@@ -153,6 +153,23 @@ def verify_package(verbose=True):
         log("✓ README is OK",
             delay=0, chevrons=False, verbose=verbose)
 
+    # Check base_payment is correct
+    config = get_config()
+    if not config.ready:
+        config.load()
+    base_pay = config.get('base_payment')
+    dollarFormat = "{:.2f}".format(base_pay)
+
+    if base_pay <= 0:
+        log("✗ base_payment must be positive value in config.txt.",
+            delay=0, chevrons=False, verbose=verbose)
+        return False
```

Comment:
If you change this to `is_passing = False`, then it will proceed with the rest of the checks, which is useful in case multiple issues are found.